### PR TITLE
Release v2.9.0

### PR DESCRIPTION
**Pull request by manatarms**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -35,5 +35,5 @@
     "locale": "en",
     "integrationName": "frontend",
     "useHostAPILogin": true,
-    "version": "2.8.0"
+    "version": "2.9.0"
```
Comment (thellimist) [inline config.js]: Version number should be `2.8.1`

**Comment by thellimist** [comment]
Could you please test every feature in WordPress before releasing. For Home Settings and More Settings, one of each type of button should satisfy.

File: config.js (hunk 1 above)
Comment (manatarms) [inline config.js]: We're adding a feature so I think it should be 2.9.0. I dont feel like this is a bug fix. But I can change it if you feel strongly.

**Comment by manatarms** [comment]
Yup I tested the front end on wordpress. But @jwineman is working on figuring the CSRF bug.

**Comment by manatarms** [comment]
Closed in favor of #137

**Pull request closed without merging**